pkg/node/node.go

```diff
@@ -357,22 +357,22 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 		return nil, fmt.Errorf("swarm key: %w", err)
 	}
 
-	psss := pss.New(swarmPrivateKey, logger)
-	b.pssCloser = psss
+	pss := pss.New(swarmPrivateKey, logger)
```

Comment:
I do not think it is good idea to rename this variable to `pss`.

Even though it might be nicer to have it like that, package with the same name is used here, and prevent any further use of that package. While in this instance there might not be any further need for something from `pss` package, we should not override name. I vote to maybe have this as guideline for Go code.

Reply:
@zelig wanted this renamed and I second that. I don't see a problem with having a variable with the same name like the package. If the compiler allows it then its probably legal no? @janos @zelig would be happy to hear your opinion.

Reply:
The problem with shadowing package name with a variable is in maintainability, as if the package is needed later in the code and time, a larger changeset is needed in order to rename the variable.

In this particular case, the last `s` in `psss` may refer to the Service, or to name the pss instance to `pssService` for clarity.

Even the compiler allows this type of shadowing, maybe there is a linter that warns about this. I would go with @zbiljic's suggestion to have a guideline.